src/ocean/core/BitArray.d

```diff
@@ -86,11 +86,13 @@ struct BitArray
                 uint[] buf = ptr[0 .. olddim];
 
                 buf.length = newdim; // realloc
+                enableStomping(buf);
+
                 ptr = buf.ptr;
-                if( newdim & 31 )
+                if( auto pad_bits = newlen & 31 )
```

Comment:
I always need to read these types of expressions twice, especially given the differences in grouping between `&` and `&&`. Is there a way to make the assignment and bitwise AND more explicit? Meaning - it's subtle difference if the `pad_bits` contains new length or just the value of the 31st bit.